src/org/labkey/response/ResponseModule.java

```diff
@@ -82,8 +86,8 @@ public boolean hasScripts()
     @NotNull
     protected Collection<WebPartFactory> createWebPartFactories()
     {
-        SimpleWebPartFactory studySetupFactory = new SimpleWebPartFactory("MyStudies Study Setup",
-            WebPartFactory.LOCATION_BODY, StudyConfigWebPart.class, null);
+        SimpleWebPartFactory studySetupFactory = new SimpleWebPartFactory("MyStudies Response Server",
+            WebPartFactory.LOCATION_BODY, MyStudiesResponseServerWebPart.class, null);
```

Comment:
You should add a legacy name below, "MyStudies Study Setup". This maintains backward compatibility with existing webparts on the page or in a folder archive.

Reply:
Resolved!